pkg/apis/noobaa/v1alpha1/noobaaaccount_types.go

```diff
@@ -74,10 +74,11 @@ type NooBaaAccountSpec struct {
 
 // AccountNsfsConfig is the configuration of NSFS of CreateAccountParams
 type AccountNsfsConfig struct {
-	UID            int    `json:"uid"`
-	GID            int    `json:"gid"`
-	NewBucketsPath string `json:"new_buckets_path"`
-	NsfsOnly       bool   `json:"nsfs_only"`
+	UID               int    `json:"uid,omitempty"`
```

Comment:
Why this is needed? If I understand correctly we won't create a CR account (we will create only the internal db account entity)